### ISSUE TITLE
Maybe bug in BelongsToManyAssociation custom_default_value

### ISSUE THREAD

**Issue by lasseebert** [crashtech/torque-postgresql#92]
Hi!

I can confirm that [my previous issue](https://github.com/crashtech/torque-postgresql/issues/91) is indeed solved in my case :tada: 

I ran into another thing now trying to implement belongs-to-many relations.

The private `custom_default_value` method fails with `undefined method 'default' for nil` on [this line](https://github.com/crashtech/torque-postgresql/blob/master/lib/torque/postgresql/associations/belongs_to_many_association.rb#L192-L194):

```ruby
def column_default_value
  owner.class.columns_hash[source_attr].default
end
```

When I debug, it turns out that `source_attr` is a symbol but `columns_hash` has string keys.

I'm not sure if the fix just is to `to_s` the symbol or if it should be changed deeper in the code. (Just changing to use `to_s` locally makes my code work).

The line in my code that triggers this, is when I try to assign to the association with something like `my_model.tags = [my_tag]`, but I guess the line in question could be reached in many way.

**Comment by crashtech** [comment]
Hey Lasse. Thanks for reporting this issue. It should be fixed on versions `3.3.3` and `2.4.4`.

**Comment by lasseebert** [comment]
Thanks for the quick fix again! :tada: 

I will test it out and let you know.
